app/src/main/java/org/simple/clinic/newentry/PatientEntryEffectHandler.kt

```diff
@@ -44,6 +44,7 @@ class PatientEntryEffectHandler(
     private val patientRepository: PatientRepository,
     private val patientRegisteredCount: Preference<Int>,
     private val ui: PatientEntryUi,
+    private val validationActions: PatientEntryValidationActions,
```

Comment:
I am assuming that the future goal is to have this interface take in a single method that has a `Field[]` as a parameter, and it's up to the implementation to decide how to show/hide the errors for a field.

Otherwise, this doesn't give us that much benefit.

Reply:
Yes, that is correct. Since the validation logic is going to become complicated. We can have a validation renderer component that just takes care of showing and hiding errors. This is a reason why we are moving in this direction.